themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ca.properties

```diff
@@ -62,6 +62,9 @@ i18n-enabled=Internacionalitzaci\u00F3 activa
 supported-locales=Idiomes suportats
 supported-locales.placeholder=Indica l''idioma i prem Intro
 default-locale=Idioma per defecte
+#localization-upload-file=Upload localization JSON file
```

Comment:
Do we need the lines at all ? The same goes for changes in other property files.

Reply:
I thought it might be helpful for someone who will translate missing localization keys. If there are in the file, he / she does not need to search for the keys in the English version.